.healthcheck.env

```diff
@@ -1,5 +1,6 @@
-# Default Template
-X_AUTHELIA_HEALTHCHECK_SCHEME=http
+# Written by Authelia Process
+X_AUTHELIA_HEALTHCHECK=1
+X_AUTHELIA_HEALTHCHECK_SCHEME=https
```

Comment:
Looks like this snuck in to your changes, probably something we have to do in the workflow to prevent it but I'm not sure what.